docs/how-to/integrations/machine-translation.md

```diff
@@ -124,10 +124,14 @@ WAGTAILLOCALIZE_MACHINE_TRANSLATOR = {
     "CLASS": "wagtail_localize.machine_translators.deepl.DeepLTranslator",
     "OPTIONS": {
         "AUTH_KEY": "<Your DeepL key here>",
+        # Optional DeepL API option. Accepts "default", "prefer_more" or "prefer_less". For more information visit the DeepL API docs link below.
+        "FORMALITY": "<Your DeepL formality preference here>",
     },
 }
 ```
 
+[DeepL API docs](https://www.deepl.com/docs-api/translate-text/)
```

Comment:
I think this should just go in the example code comment. Will tidy up when I come to merge

Reply:
Sounds good, did it like this because hyperlinks don't format properly inside the python block